presto-cassandra/src/test/java/com/facebook/presto/cassandra/MockCassandraSession.java

```diff
@@ -124,35 +116,18 @@ public CassandraTable getTable(SchemaTableName tableName)
     @Override
     public List<CassandraPartition> getPartitions(CassandraTable table, List<Object> filterPrefix)
     {
-        accessCount.incrementAndGet();
-        if (throwException) {
-            throw new IllegalStateException();
-        }
-
-        return super.getPartitions(table, filterPrefix);
+        throw new UnsupportedOperationException();
     }
 
     @Override
-    protected List<Row> queryPartitionKeys(CassandraTable table, List<Object> filterPrefix)
```

Comment:
This probably should be a separate commit, to ensure that this commit is purely an automatic extraction of interface.
I won't insist if it's too much hassle to split it.